modules/ROOT/nav.adoc

```diff
@@ -117,6 +117,7 @@
    **** xref:n1ql:n1ql-language-reference/metafun.adoc[Miscellaneous Utility Functions]
     ***** xref:n1ql:n1ql-language-reference/advisor.adoc[ADVISOR Function]
     ***** xref:n1ql:n1ql-language-reference/curl.adoc[CURL Function]
+    ***** xref:n1ql:n1ql-language-reference/unnestposfun.adoc[UNNEST_POS Function]
```

Comment:
Don't think this function needs a page by itself? CURL() and ADVISOR() do need a page each, because they are massive, but this is just a pretty straighforward function. Happy to classify this as under "miscellaneous" but I would add it to `metafun.adoc`.